specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -5331,6 +5450,135 @@
         }
       }
     },
+    "IpAddressAggregate": {
+      "description": "The key value pair for properties.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Unique ID for the entree",
+          "type": "string"
+        },
+        "tenantId": {
+          "description": "The tenant ID",
+          "type": "string"
+        },
+        "serviceId": {
+          "description": "The service ID",
+          "type": "string"
+        },
+        "ipAddress": {
+          "description": "The Bad Ip Address",
+          "type": "string"
+        },
+        "timestamp": {
+          "description": "When the event occurred",
+          "type": "string"
+        },
+        "firstAuditTimestamp": {
+          "description": "The first audit timestamp",
+          "type": "string"
+        },
+        "lastAuditTimestamp": {
+          "description": "The last audit timestamp",
+          "type": "string"
+        },
+        "extranetLockoutErrorCount": {
+          "description": "The extranet lockout error count",
+          "type": "integer"
+        },
+        "badPasswordErrorCount": {
+          "description": "The bad password error count",
+          "type": "integer"
+        },
+        "uniqueUsernamesAttemptedCount": {
+          "description": "The unique usernames attempted",
+          "type": "integer"
+        },
+        "attemptCountThresholdIsExceeded": {
+          "description": "Has the attempt count threshold been exceeded",
+          "type": "boolean"
+        },
+        "timeSpan": {
+          "description": "The timespan of the event",
+          "type": "string"
+        },
+        "isWhitelistedIpAddress": {
+          "description": "Is the address a whitelisted IP address?",
```

Comment:
Please scrub all descriptions for inconsistent casing for IP. It should all be upper case. Also here the standard .NET (Maybe stylecop) phrase sounds better. "**A value indicating whether** the IP address has been whitelisted." This can be used for all boolean type descriptions.

Reply:
Fixed